homeassistant/components/unifi/__init__.py

```diff
@@ -64,6 +66,21 @@ async def async_unload_entry(hass, config_entry):
     return await controller.async_reset()
 
 
+async def async_migrate_entry(hass, config_entry):
+    """Migrate old entry."""
+    LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    #  Flatten configuration but keep old data if user rollbacks HASS prior to 2021.03
```

Comment:
Maybe mention in the comment that the version is intentionally not bumped to be backwards compatible.